src/reanimated2/shareableCache.ts

```diff
@@ -0,0 +1,7 @@
+'use strict';
+import type { ShareableRef } from './commonTypes';
+
+export const _shareableCache = new WeakMap<
```

Comment:
I think it could be a good idea to place here a short comment why we put this register to separate file.